packages/visual-editor/src/hooks/useEditorSubscriber.ts

```diff
@@ -281,6 +281,7 @@ export function useEditorSubscriber() {
         }
         case INCOMING_EVENTS.ComponentDragStarted: {
           dragState.updateIsDragStartedOnParent(true);
+          setDraggingOnCanvas(true);
```

Comment:
Testing this out locally, I really like this change to trigger the drag sooner (as soon as the card is picked up)